### ISSUE TITLE
Clicking on tags brings up broken XML page

### ISSUE THREAD

**Issue by Darthagnon** [g1eny0ung/hugo-theme-dream#264]
Both on [your blog](https://g1en.site) and [mine](https://robot-one.github.io), clicking on a tag in the tag cloud or on the All Tags page will break the webpage.

**Comment by techmagus** [comment]
@Darthagnon 

It probably is the empty file `layouts/tags/term.html` (ece0e00) that is causing it. Try to remove it in your copy of the theme, it should work then.

You can check the tags on my blog here: https://im.youronly.one/yuki/tags or an example tag https://im.youronly.one/yuki/tag/explained/

**Comment by Darthagnon** [comment]
@techmagus Thank you for the clarification, [someone said the same on the Hugo forums](https://discourse.gohugo.io/t/site-tags-rendering-as-xml-not-html/38650/). I've been working on what is more-or-less [a hard fork of this theme](https://github.com/Darthagnon/hugo-theme-dream/tree/production) - do you have the source for your blog's version of the theme, or did you just replace "tags" with "categories"? What you've got on your blog is exactly what I was dreaming of doing on [my website](https://robot-one.github.io/) (GitHub/GH Pages seems to be having connectivity issues at the moment). If you could share your source, or if you feel like opening a pull request on my fork, I'll merge it immediately.

**Comment by Darthagnon** [comment]
@g1eny0ung Doesn't seem to be much maintenance going on on this theme, any chance you could please disable these annoying bots from closing issues?

**Comment by g1eny0ung** [comment]
@Darthagnon I've realized this problem...Sorry about this, I don't have the energy to maintain this theme for now, and I may have to leave for a while because I have other things to do. I will remove the stale bot and recover the unsolved issues.

**Comment by Darthagnon** [comment]
> @Darthagnon I've realized this problem...Sorry about this, I don't have the energy to maintain this theme for now, and I may have to leave for a while because I have other things to do. I will remove the stale bot and recover the unsolved issues.

No worries; I know life gets in the way. Thank you for developing it in the first place! I'm currently maintaining the theme in [my fork](https://github.com/Darthagnon/hugo-theme-dream), since I actively use it on my homepage. I'll open an issue to link to it. Maybe eventually we can merge them again.

**Comment by techmagus** [comment]
> do you have the source for your blog's version of the theme, or did you just replace "tags" with "categories"?

I have, but it's only one change in this fork: https://github.com/techmagus/hugo-theme-dream

The change for the tags/categories is in my website repo: https://github.com/YourOnly-One/site.yuki (mirror). But, yes, I just changed it to categories. I think it was this file: https://github.com/YourOnly-One/site.yuki/blob/main/layouts/partials/header.html

> PS. if you want to colab on developing this theme further, I'm all ears.

I'll check my load. Though, I've been meaning to submit a PR for the translation bit since I started using the theme, it's 2023 and I still haven't. ^_^;; In my site, there are flags at the top bar, and when viewing a post, there are translations on the sidebar, too. (My website repos are a mess, since I use it for testing stuff and what not.)

> I don't have the energy to maintain this theme for now, and I may have to leave for a while because I have other things to do.

No worries about that! That's where the community comes in, and the beauty of FLOSS/FOSS.
